app/shared/components/Wallet/Modal/Account/Lookup.js

```diff
@@ -85,11 +85,10 @@ export default class WalletModalAccountLookup extends Component<Props> {
                         {t('wallet_account_lookup_modal_accounts_found')}
                       </Header>
                       <List divided relaxed>
-                        {(!__lookups.length)
+                        {(!__lookups || !__lookups.length)
```

Comment:
I'm not clear on why this is needed if `_lookups` is getting set on initial state and reset, I'm probably missing something though 😄

Reply:
I am not honestly sure either, but it's happening. It could be that the first time the state is being generated it's calling an API (which has a delay), and the UI is rendering before that. It also could be a state from an older version of the application that was cached and didn't have it. 

You're probably right in assuming that the real problem here is that the state isn't getting initialized before the UI - I'm just not sure how to fix that immediately :)